ui/pages/permissions-connect/snaps/snap-install/index.scss

```diff
@@ -15,7 +16,7 @@
     position: sticky;
     right: 0;
     left: 0;
-    bottom: 0;
+    bottom: -16px; // accounts for snap-install__content padding
```

Comment:
Would be nice to figure out a way to add this into snap-install__content, but I assume that it's not as easy as it sounds 👍

Reply:
it's true. seems not to be that easy 🥲  I removed the fixed position in the last PR which broke instances. the snap footer has instances which it is a sibling to the snap content and other instances where it's a child of the snap content. Unless the snap footer is consistently a sibling of the content, we're quite limited in solutions while we're dependent on the footer being position fixed.

Reply:
actually, I forgot I removed top and bottom padding from the content here https://github.com/MetaMask/metamask-extension/pull/31454/commits/3a6e18956694b8ce58d8d685b8551bb5ba6a6f1c 

which means we can remove these. thanks for the catch! Updated https://github.com/MetaMask/metamask-extension/pull/31454/commits/305e3e909936e12f819bf0b846a7fe62d3a36d86

cc: @FrederikBolding